Savanna/View/UserInterface.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Savanna
+{
+    public class UserInterface
+    {
+        /// <summary>
+        /// Makes view of the game field.
+        /// </summary>
+        /// <param name="args">Array containing the game field.</param>
+        /// <param name="game">Current game generation.</param>
+        public static void Display(string[,] args, Game game)
```

Comment:
Generic name. Here and in other places replace with more descriptive one. May be:
gameField

Reply:
Renamed.